### PR TITLE
Possible change to PosDiff

### PR DESCRIPTION
**Pull request by zhut19**
This is an attempt to make the PosDiff cut a bit more reasonable, by 1) Integrating R square into D square cut. 2) Add S2 area dependence to the D square upper limit.

### CODE REVIEW

**Comment by zhut19** [comment]
To visualize the old cut, here we reparameterized the data, into the figure following, that we have R square cut in black lines, and D square cut in red line. And we realized 

A) The R square cut is cutting into signals at high radius, while overlapping D square cut in low radius.
B) The nn is systematically constructing outwards, and the radial distance from tpf is oscillating.
pos_diff_0

**Comment by zhut19** [comment]
One obvious improvement is to remove R square cut and recenter the Rnn - Rtpf to center around 0 by subtracting the purple line in figure above. Thus we can get the plot below.
pos_diff_1

**Comment by zhut19** [comment]
Once we reduce the position parameter to just adjusted distance between nn tpf position, we can look at the S2 area dependence clearly, and only cut on D and S2 area space. The result is following, also we examine the same Rn220 leakage events we saw before shown in [note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:analysis:sr1:pos_cut_v0).

pos_diff_2

pos_diff_5

File: lax/lichens/sciencerun0.py [hunk 1]
```diff
@@ -814,12 +814,15 @@ class PosDiff(Lichen):
     which can partly remove wall leakage events due to the small size of S2.
     Contact: Yuehuan Wei <ywei@physics.ucsd.edu>
```
Comment (pdeperio) [inline lax/lichens/sciencerun0.py]: Add your contact info and new Wiki note link.

**Review by pdeperio — CHANGES_REQUESTED** [review]

**Review by pdeperio — APPROVED** [review]

**Pull request merged**